src/configuration/GPG/GnuPG/params.txt

```diff
@@ -0,0 +1,9 @@
+Key-Type: RSA
+Key-Length: 8192  
+Subkey-Type: RSA
+Subkey-Length: 8192
```

Comment:
Because it's way overkill. The French Cybersecurity agency (ANSSI) advise a minimal size of 3072 bits for post-2030 key usage. 

It's on par with other recommendations (NIST, NSA, for example). Also, the GPG FAQ explains really well why 4096+ bits key size are generally useless : https://www.gnupg.org/faq/gnupg-faq.html#no_default_of_rsa4096